userguide/system.rst

```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
   diagnosing a performance bottleneck.
 
 * **Environment** is a drop-down menu to indicate the role of the
-  affected system. Choices are *Production*, *Staging*, *Test*,
-  *Prototyping*, or *Initial Deployment/Setup*.
+  affected system.
+
+  .. tabularcolumns:: |>{\RaggedRight}p{\dimexpr 0.20\linewidth-2\tabcolsep}
+                      |>{\RaggedRight}p{\dimexpr 0.16\linewidth-2\tabcolsep}
+
+.. _environment options:
+
+.. table:: Environment Options
+      :class: longtable
+
+   +---------------------+----------------------------------------------------------+
+   | Environment         | Description                                              |
+   +=====================+==========================================================+
+   | Production          | The system is being used for production. Data loss       |
+   |                     | affects customers directly.                              |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Staging             | The system is being prepared for production.             |
+   |                     |                                                          |
+   +---------------------+----------------------------------------------------------+
+   | Test                | The system is being used internally.                     |
+   |                     | The system does not affect customers.                    |
+   |                     |                                                          |
```

Comment:
The sentence for the "Test" section is ambiguous and confusing, imo. My recommendation is to saying something like `The system is being used internally for testing purposes.` Remove the sentence `The system does not affect customers.`